File: code/Templates/PythonModel/mymodel.py
import numpy as np


num_params = 10

def prior_transform(us):
    return us - 0.5

def log_likelihood(params):
    return -num_params*0.5*np.log(2*np.pi*0.01**2)-0.5*np.sum((params/0.01)**2)


def both(us):
    return log_likelihood(prior_transform(us))

File: code/Templates/JuliaModel/mymodel.jl
num_params = 10


function prior_transform(us)
    return (us .- 0.5)
end

function log_likelihood(xs)
    return (-num_params*0.5*log(2*pi*0.01^2) - 0.5*sum((xs/0.01).^2))
end

function both(us)
    return log_likelihood(prior_transform(us))
end

